templates/dispatch.html

```diff
@@ -2,16 +2,142 @@
 <html lang="en">
 <head>
   <meta name="viewport" content="width=device-width" />
-  <link rel="stylesheet" href="{{ url_for('static', filename='styles/style.css') }}"/>
-  <link rel="shortcut icon" href="{{ url_for('static', filename='image/favicon.ico') }}"/>
-  <title>Dispatch | SourceCAD</title>
+  <link rel="shortcut icon" href="image/favicon.ico" />
```

Comment:
Same here, you've removed `"{{ url_for('static', filename='image/favicon.ico') }"}` and the like. Please fix.